src/ai_hawk/job_manager.py

```diff
@@ -500,15 +490,21 @@ def job_tile_to_job(self, job_tile) -> Job:
             logger.warning(f"Failed to extract job ID: {e}", exc_info=True)
 
         try:
-            job.location = job_tile.find_element(By.XPATH, './/span[contains(normalize-space(), " · ")]').text.split(' · ')[-1].strip()
+            job.location = job_tile.find_element(By.CLASS_NAME, 'job-card-container__metadata-item').text
```

Comment:
Simply doesn't work. If we check, 'job-card-container__metadata-item' never follow as to location, but to salary fork. Has to be reverted with no doubt